proprietary-files.txt

```diff
@@ -25,7 +25,7 @@ vendor/firmware/bcm43438A1_V0051.0217.hcd
 # Camera
 ###################
 
-lib/hw/camera.universal7580.so
+lib/hw/camera.vendor.universal7580.so
```

Comment:
This is not correct. The proprietary-files.txt is what is dumped from a vendor image (or current lineageos image).

As we currently rename this blob so we can use my camera wrapper this is probably more suitable:
lib/hw/camera.universal7580.so:lib/hw/camera.vendor.universal7580.so

Reply:
Thanks for the information